reana_commons/models.py

```diff
@@ -169,6 +169,8 @@ class Job(Base, Timestamp):
     max_restart_count = Column(Integer)
     deleted = Column(Boolean)
     logs = Column(String, nullable=True)
+    prettified_cmd = Column(String(1000))
+    step_name = Column(String(256))
```

Comment:
Even though step is used everywhere else in yadage, cwl and even serial, I would call this just `name` since job's don't know about workflows or workflow specifications and it should remain as this (workflow_uuid is the exception)